con-base/src/scripts/db/concierge_database.sql

```diff
@@ -1 +1,62 @@
 CREATE SCHEMA `concierge` DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci ;
+
+DROP TABLE IF EXISTS `concierge`.`continents`;
```

Comment:
Why we are checking if table 'continents' exists and than create a table 'conutries'?